corehq/apps/fixtures/views.py

```diff
@@ -567,7 +591,11 @@ def _get_fixture_upload_args_from_request(request, domain):
             "User {} doesn't have permission to upload fixtures"
             .format(request.couch_user.username))
 
-    return _excel_upload_file(upload_file), replace, is_async
+    skip_orm = False
+    if request.POST.get('skip_orm') == 'true' and SKIP_ORM_FIXTURE_UPLOAD.enabled(domain):
```

Comment:
I don't see where `skip_orm` is added to a form anywhere. Is that intentional?

Reply:
Correct. this is only going to be for an API call as that's the only way their using it